build/main.go

```diff
@@ -247,6 +260,8 @@ func main() {
 		buildGengen()
 	case "generate-genesis":
 		generateGenesis()
+	case "build-migrations":
```

Comment:
I am good with not adding `install-migrations` (it matches up with other tool options).  Just checking that you made the choice to leave it off explicitly as install shows up as an optional acceptance criteria in 2582.

Reply:
thanks, it was intentional.